app/models/current_order.rb

```diff
@@ -58,7 +61,7 @@ def paypal_url(current_user, return_url)
 
   def save_order(current_user)
     user = current_user
-    new_order = user.orders.new(:total => @total, :vat => vat, :delivery_cost => @delivery_cost, :invoice => @invoice, :Status => @status, :transaction_id => @transaction_id)
+    new_order = user.orders.new(:total => @total, :vat => vat, :delivery_cost => @delivery_cost, :invoice => @invoice, :Status => @status, :transaction_id => @transaction_id, :pickup_time => @pickup_time)  
```

Comment:
Line is too long. [206/80]<br>Use the new Ruby 1.9 hash syntax.<br>Trailing whitespace detected.